src/AvtaleSide/NesteForrige/NesteForrige.less

```diff
@@ -0,0 +1,23 @@
+.nesteforrige {
+    display: flex;
+    flex-direction: column;
+    position: sticky;
+    top: 0;
+
+    &__lenke {
+        width: 100%;
+        margin-bottom: 1rem;
+        word-break: break-all;
+    }
+    &__buttontoright {
+        margin-bottom: 1rem;
+        margin-inside: 1rem;
+        white-space: unset;
+        /*align-items: stretch;*/
+        margin-left: 1rem;
+        float: right;
```

Comment:
Ikke kombiner float og flex

Reply:
Kan jeg bruke kun float? Funker ikke med meg med flex :(

Reply:
Sjekk ut https://css-tricks.com/snippets/css/a-guide-to-flexbox/ og se på `justify-content: space-between`. Den gjør det du vil oppnå.

Reply:
Takk, den funker nå.